app/views/admin/tickets/_form.html.haml

```diff
@@ -12,7 +12,10 @@
       = f.input :title, input_html: { autofocus: true }
       = f.input :description, input_html: { rows: 5, data: { provide: 'markdown-editable' } }
       = f.input :price
-      = f.input :price_currency, as: :select, class: 'form-control', collection: ['USD', 'EUR', 'GBP', 'INR', 'CNY', 'CHF'], include_blank: false
+      - if (@conference.tickets.count >= 1 && @ticket.new_record?) || @conference.tickets.count > 1
```

Comment:
I am not sure if I follow this condition, why isn't it enough with `@conference.tickets.count > 1` `count` raises a query to the database so new records doesn't show up.

Reply:
With only @conference.tickets.count > 1 the second added ticket would still be able to change the currency. @conference.tickets.count >= 1 && @ticket.new_record? makes sure that you can still change currency if you're editing the first ticket (I went on the assumption that was desired, but now I see it isn't).